### ISSUE TITLE
Lidarr - Unknown scan request from

### ISSUE THREAD

**Issue by zapras** [l3uddz/plex_autoscan#127]
Lidarr doesn't seem to work with Plex. It works fine with Radarr/Sonarr. I've even managed to get the manual scan working with my Music library, but can't get it work from Lidarr.

Only seems to happen on Lidarr Webhook events. 

` 2019-11-25 19:57:11,290 -    ERROR -  AUTOSCAN [140706736985856]: Unknown scan request from: '172.18.0.17'`

I've updated to the latest Plex_Autoscan Master version.
Docker: linuxserver/lidarr 

My Radarr/Sonarr docker containers are using hotio. Not sure if that makes a difference?

Looks very similar to [this](https://github.com/l3uddz/plex_autoscan/issues/89l) issue. I've already updated plex_autoscan, not sure what else to do?

**Comment by NBSDEVApp** [comment]
Exactly the same bug

**Comment by hotio** [comment]
I can confirm this, lidarr v0.7.1.1652, plex_autoscan 4e31fb19d81ca9d7ff0fc2f362f9accfff979bc4

**Comment by tamburra** [comment]
Yes this does seem to be an issue I am having as well currently.

**Comment by Ptikrazy** [comment]
I'm on Lidarr v1.0.0.2263 and currently this issue still exists.
